autonomi/Cargo.toml

```diff
@@ -23,7 +23,6 @@ name = "put_and_dir_upload"
 default = []
 external-signer = ["ant-evm/external-signer"]
 extension-module = ["pyo3/extension-module"]
-loud = []
```

Comment:
This feature is very useful for local testing for us, I think we should keep it around. Most users will never us it but that's fine, it's off by default.